keycloak-gatekeeper/templates/pdb.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.pdb.enabled }}
-apiVersion: policy/v1beta1
```

Comment:
curious is this version deprecated?

Reply:
Deprecated in 1.21, removed in 1.25
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125